server/package.json

```diff
@@ -56,7 +57,7 @@
     "serve-favicon": "~2.4.2",
     "slack": "latest",
     "socket.io": "^2.0.2",
-    "sqlite3": "~4.0.2",
+    "sqlite3": "^5.1.7",
```

Comment:
Does it still run the tests? Some sqlite changes are breaking!

Reply:
I have just done a fresh setup and seems to be running with no issues

Reply:
Okay, tests via github actions currently fail because they were pulled from the master branch, which still tries to use node 12. We'll have to re-try the current test suite for a usable result.